### DISCUSSION TITLE
Is it possible to sort buffers list alphabetically?

### DISCUSSION THREAD

**Discussion by niamleeson** [ibhagwan/fzf-lua#1179] [Q&A] [answered]
It would be great to sort the list by file name (not path) so I know where it is immediately. The frequency works well, but sometimes I lose track of a file because it was moved due to frequency.

**Comment by ibhagwan** [comment, marked as answer]
Not alphabetically but by buffer number is possible if you disable the last used sorter with `:FzfLua buffers sort_lastused=false`
